### DISCUSSION TITLE
Change a Caution message to be more useful in standard logging level

### DISCUSSION THREAD

**Discussion by fellipec** [abraunegg/onedrive#2993] [General]
When I use the software with the regular log level, sometimes I get messages like this:

`CAUTION The JSON element transmitted by the Microsoft OneDrive API includes HTML URL encoded items`

But in the log there is nowhere to find the file that cause the error, unless I add `--verbose` to the command line.

The downside of doing this is that the logs became huge. I don't need this level of logging all the time. Can I get the filename responsible for the error in the standard log level?

**Comment by abraunegg** [comment]
The offending file should the file name attempted to be downloaded ... will look into it
